omero/sysadmins/unix/install-web/install-nginx.txt

```diff
@@ -282,10 +244,6 @@ using :property:`omero.web.wsgi_args`:
 EXPERIMENTAL: Gunicorn advanced configuration
 ---------------------------------------------
 
-.. note:: Experimental configurations are not ready for production use.
```

Comment:
What is the reasoning behind leaving these subtitles as 'Experimental' but removing the note boxes explaining what that means? Would it be better to reinstate the notes but rephrase them to "these configurations have yet to be extensively tested in a production environment, use at your own risk" or similar?